ocaml/idl/datamodel_types.ml

```diff
@@ -179,6 +180,12 @@ let release_order_full = [{
       version_major = 2;
       version_minor = 7;
       branding   = "XenServer 7.2";
+    }; {
+      code_name     = Some rel_inverness;
```

Comment:
Another option is to simply not add Inverness to `release_order_full` yet, and add it later.